conda-requirements.txt

```diff
@@ -14,4 +14,4 @@ blas
 gsl
 scikit-learn
 dadi
-slim=3.7.1
\ No newline at end of file
+slim=4.0
```

Comment:
worried about this move to 4.0-- is the "tick" timescale going to mess anything up? probably not as we are just doing WF models, yes?

Reply:
stdpopsim itself has moved to slim v4, so we better have this!